src/signals/incident-management/containers/IncidentDetail/components/Detail/components/ExtraProperties/index.test.js

```diff
@@ -1,67 +1,89 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render } from '@testing-library/react';
 
 import ExtraProperties from './index';
 
 describe('<ExtraProperties />', () => {
-  let wrapper;
-  const props = {
-    items: [
-      {
-        id: 'extra_straatverlichting',
-        label: 'Gaat uw melding over één of over meer lampen?',
-        category_url: '/signals/v1/public/terms/categories/wegen-verkeer-straatmeubilair/sub_categories/straatverlichting-openbare-klok',
-        answer: {
-          id: 'meer_lampen',
-          label: 'Meer lampen'
+  let props;
```

Comment:
Doesn't have to be a `let`; value doesn't change over time.

Reply:
gedaan

Reply:
gedaan